docs/using/installing/index.rst

```diff
@@ -4,17 +4,21 @@
 Installing Flocker
 ==================
 
-As a user of Flocker you will need to install the ``flocker-cli`` package which provides command line tools to control the cluster.
-This should be installed on a machine with SSH credentials to control the cluster nodes
-(e.g., if you use our Vagrant setup then the machine which is running Vagrant).
+The Flocker CLI is installed on your local machine via the ``flocker-cli`` package which provides command line tools to control the cluster. 
```

Comment:
I think that this introduces confusion between `flocker-cli` and "The Flocker CLI". As I mentioned in the glossary I think we should stick to having them be synonymous. Particularly because I don't `brew install flocker-cli` - so a Mac user will never see `flocker-cli` on their machine.